src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+    >
+      <fieldset class="form-block">
+        <legend>Personal information</legend>
+        <label class="input-block">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
```

Comment:
Please add for all text fields
![image](https://user-images.githubusercontent.com/54936044/229271769-47faa71e-96e7-44ed-b505-e67993f73727.png)